### PR TITLE
Added a review system to the my_bookings page

### PR DESCRIPTION
**Pull request by aurelien-jacomy**
On the my_booking pages: 

- Show a link to add a review if the service has been done or canceled, and if there is no review yet. I build a method add_review? to check this
- Add a link to the review already made, with the rating

image

### CODE REVIEW

File: app/views/bookings/index.html.erb [hunk 1]
```diff
@@ -10,10 +10,26 @@
     <% @my_bookings.each do |booking| %>
     <div class="card">
 
+        <%= link_to booking.service.name, booking.service  %>
```
Comment (gabantonini) [inline app/views/bookings/index.html.erb:13]: esse link funciona?
fiquei na duvida que nao tem o _path

**Comment by aurelien-jacomy** [comment]
Also added an alert all pages when there are pending reviews. I created a class method on Booking to check if there are pending reviews for a specific user. Pending = Service done during the past 7 days. 

image

File: app/views/bookings/index.html.erb (hunk 1 above)
Comment (aurelien-jacomy) [reply app/views/bookings/index.html.erb:13]: funciona. Você pode colocar uma instância em vez de colocar um path. Ele vai pro show da instância.

File: app/models/booking.rb [hunk 2]
```diff
@@ -2,4 +2,13 @@ class Booking < ApplicationRecord
   belongs_to :service
   belongs_to :user
   has_one :review, dependent: :destroy
+
+  def add_review?
+  	(["Realizado", "Cancelado"].include?(status)) & (self.review.nil?)
```
Comment (gabantonini) [inline app/models/booking.rb:7]: &&

File: app/views/bookings/index.html.erb (hunk 1 above)
Comment (victoralbanezi) [reply app/views/bookings/index.html.erb:13]: não sabia dessa também

**Comment by gabantonini** [comment]
👍

**Pull request merged**